create-doi-request/src/main/java/no/unit/nva/doirequest/create/CreateDoiRequestHandler.java

```diff
@@ -8,14 +8,14 @@
 import java.time.Clock;
```

Comment:
setAdditionalHeadersSupplier is deprecated, is there a reason for still using this/not fixing it here?